### PR TITLE
merge lexy-book-db into main

### PR DESCRIPTION
**Pull request by alexamaingard**

### CODE REVIEW

**Review by TheunisK — APPROVED** [review]

File: src/server/controllers/init.js [hunk 1]
```diff
@@ -50,16 +49,60 @@ const addBookToDB = async (req, res) => {
                         },
                     };
                 }),
-            }
+            },
```
Comment (hannah68) [inline src/server/controllers/init.js:52]: Great job Alxa

**Review by hannah68 — APPROVED** [review]

File: prisma/schema.prisma [hunk 2]
```diff
@@ -73,3 +75,11 @@ model Recommendation {
...
   updatedAt   DateTime      @updatedAt
 }
+
+model Group {
+  id          Int           @id @default(autoincrement())
+  number      Int
+  createdAt   DateTime      @default(now())
+  updatedAt   DateTime      @updatedAt
+  books       Book[]
+}
```
Comment (Can-Tina) [inline prisma/schema.prisma:85]: Looks good! :)

File: src/server/data.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+            'Man Versus the Natural World',
+            'The Home',
+            'Friendship'
+        ]
+    }
+]
+
+module.exports = {
+    BOOKS_TO_SEED
+}
```
Comment (Can-Tina) [inline src/server/data.js:175]: Impressive work!

File: src/server/routers/init.js [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 
 const {
-    addBookToDB
+    seedBookDatabase
 } = require('../controllers/init.js');
 
 const router = express.Router();
 
-router.post('/', addBookToDB);
+router.post('/', seedBookDatabase);
```
Comment (Can-Tina) [inline src/server/routers/init.js:9]: Looks good!

**Review by Can-Tina — APPROVED** [review]

File: src/server/routers/init.js (hunk 4 above)
Comment (glowkeeper) [reply src/server/routers/init.js:9]: Ooooo - I reckon this might mean '/' resolves correctly on heroku, right?

**Review by glowkeeper — APPROVED** [review]
Approved

File: src/server/routers/init.js (hunk 4 above)
Comment (alexamaingard) [reply src/server/routers/init.js:9]: I didn't take a look at the heroku thing yet. I'll take a look now and come back to you.

**Pull request merged**